src/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "nativescript-oauth2",
-  "version": "2.5.0",
+  "version": "2.5.0-rc.0",
```

Comment:
@nativescript/core and types are not released as stable yet, I would follow that naming to let the user know about upcoming updates, once stable packages are released